examples/oauth-v1/app.js

```diff
@@ -36,7 +36,7 @@ app.get('/slack/install', async (req, res, next) => {
 
 // example 1
 // empty callbackOptions Object
-// use slackOauth default success and failure methods
+// use default success and failure methods
```

Comment:
could we make a similar comment on L45 saying "using custom success and failure handlers", or something of that effect?